integration-tests/api/__tests__/admin/order.js

```diff
@@ -222,6 +221,48 @@ describe("/admin/orders", () => {
     })
   })
 
+  describe("POST /admin/orders/:id/swaps", () => {
```

Comment:
Created additional describe for /swaps as I wanted to use different seeders,

Reply:
This is so, since the swap seeder and claim seeder has some overlapping features